src/Student.java

```diff
@@ -1,15 +1,74 @@
-
+import com.sun.org.apache.xerces.internal.impl.xs.util.StringListImpl;
+import com.sun.org.apache.xerces.internal.xs.StringList;
 
 public class Student {
-    String nameStudent;
-    String gradeEnter;
-    String matterRecorded;
+    String name;
+    String grade;
+    String subject;
+
+    public Student(String name, String grade, String subject){
+        this.name = name;
+        this.grade = grade;
+        this.subject = subject;
+    }
 
-    public Student(String nameStudent, String gradeEnter, String matterRecorded){
-        this.nameStudent = nameStudent;
-        this.gradeEnter = gradeEnter;
-        this.matterRecorded = matterRecorded;
+    public void printStudent(int i){System.out.println(i+1+". Student's name "+this.name +" "+"grade you are entering is "+this.grade +" "+"your registered subjects are "+this.subject);}
+    public void printGrade(int i){System.out.println(i+1+". Student's name "+this.name +" "+"grade you are entering is "+this.grade);}
+    public void printSubject(int i){System.out.println(i+1+". Student's name "+this.name +" "+"your registered subjects are "+this.subject);}
+
+    public boolean isGrade(){
+        if (grade.contains("sixth")){
```

Comment:
This is too much work, you should better use an int or enum as data type for the grades